ui/src/pages/Dashboard/NodeInfo/Node.vue

```diff
@@ -188,16 +277,55 @@ export default {
       return this.pagination.perPage * (this.pagination.currentPage - 1);
     },
     total() {
-      this.paginationTotal(this.tableData.length);
-      return this.tableData.length;
+      let result;
+      if (this.searchQuery !== '') {
+        const temp = [];
+        result = this.fuseSearch.search(`=${this.searchQuery}`);
+        for (let i = 0; i < Object.keys(result).length; i += 1) {
+          temp.push(result[i].item);
+        }
+        result = temp;
+      } else if (this.filters.default === 'node version' && this.filterssign.default === '>=') {
+        result = this.filter1;
+      } else if (this.filters.default === 'node version' && this.filterssign.default === '<') {
+        result = this.filter3;
+      } else if (this.filters.default === 'nodes hashes' && this.filterssign.default === '>=') {
+        result = this.filter2;
+      } else if (this.filters.default === 'nodes hashes' && this.filterssign.default === '<') {
+        result = this.filter4;
+      } else {
+        result = this.tableData;
+      }
+      this.paginationTotal(result.length);
+      return result.length;
     },
   },
   mounted() {
     this.isLoading = true;
     axios
-      .get('https://stats.runonflux.io/fluxinfo?projection=node')
+      .get('https://stats.runonflux.io/fluxinfo?projection=node,flux,appsHashesTotal')
```

Comment:
move to separate async method, Use try catch block with const data = await axios.get instead of promise for better redability

Reply:
let me check if this is possible async in mounted

Reply:
This does not have to be in mounted. In mounted you will call
this.myAsyncMethod()
and then you have 
async myAsyncmethod() {...

Reply:
Let me try this one. thanks